x/gov/tick.go

```diff
@@ -0,0 +1,144 @@
+package gov
+
+import (
+	"fmt"
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// Called every block, process inflation, update validator set
+func EndBlocker(ctx sdk.Context, keeper Keeper) {
```

Comment:
overall, I'd eliminate tick by moving `EndBlocker` to `handler.go` and have tally be it's own file `tally.go` - think this could make it clearer - not vital for now though